index.html

```diff
@@ -7,10 +7,99 @@
+    <link rel="stylesheet" href="style.css" />
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <div class="content">
+        <header>
+          <div class="logo">
+            <img class="logo-img" src="image/cake1.jpeg" alt="logo" />
+          </div>
+          <div>
+            <img src="image/hamburger.jpg" class="hamburger-menu" />
+          </div>
+          <p><i><b>The best cakes in town</b></i></p>
+          <p><i><b>delivered to your door</b></i></p>
+        </header>
+        <nav>
+          <ul class="navbar">
+            <li class="nav-item">
+              <a class="nav-link" href="#">HOME</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">CAKES</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link">ORDERING</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Delivery</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">ABOUT</a>
+            </li>
+          </ul>
+        </nav>
+        <main>
+          <div class="intro">
+            <div class="wlc">
+              <p>Welcome</p>
+              <p>
+                Dear customer, Welcome to Delicious cake in world. We
+                are the UK's leading cake decorating and sugar craft store and
+                specialists in baking,decorating celebration cakes. Choose your perfect cake, buy online and get free
+                delivery.
+              </p>
+            </div>
+            <div class="big-img">
+              <img src="/image/cake2.jpeg" alt="cake2" />
+            </div>
+          </div>
+          <div class="products">
+            <div class="cake1-img">
+              <img src="/image/cake3.jpeg" alt="cake3" />
+            </div>
+            <div class="cake2-img">
+              <img src="/image/cake4.jpg" alt="cake4" />
+            </div>
+  
+            <div class="cake3-img">
+              <img src="/image/cake5.jpeg" alt="cake5" />
+            </div>
+            <div class="cake4-img">
+              <img src="/image/cake6.jpg" alt="cake6" />
+            </div>
+          </div>
+        </main>
+        <footer>
+          <div class="footer-content">
+            <div class="social-media">
+              <a href="#"
+                ><img src="image/twiter.jpeg" alt="Twitter icon"
+              /></a>
+  
+              <p class="contact-us">Contact us</p>
+            </div>
+            <div class="social-media">
+              <a href="#"
+                ><img src="image/facebook.png" alt="Facebook icon"
+              /></a>
+  
+              <p class="contact-us">Join us</p>
+            </div>
+            <div class="social-media">
+              <a href="#"
+                ><img src="image/inestagram.jpg" alt="Instagram icon"
+              /></a>
+</body>   <p class="contact-us">Follow us</p>
```

Comment:
Don't forget a proper indentation for your HTML code (any code you write really). This is importaint for reader's experience.